.github/ISSUE_TEMPLATE/comp_exporter_geneva.md

```diff
@@ -0,0 +1,42 @@
+---
+name: OpenTelemetry.Exporter.Geneva
+about: Issue with OpenTelemetry.Exporter.Geneva
+labels: comp:exporter.geneva
```

Comment:
FYI, I was stealing the pattern from https://github.com/open-telemetry/opentelemetry-java-contrib/issues.